Sources/Calendar.php

```diff
@@ -365,6 +423,55 @@ function CalendarPost()
 	$context['linktree'][] = array(
 		'name' => $context['page_title'],
 	);
+
+	loadCSSFile('jquery-ui.datepicker.css', array('defer' => false), 'smf_datepicker');
+	loadCSSFile('jquery.timepicker.css', array('defer' => false), 'smf_timepicker');
+	loadJavascriptFile('jquery-ui.datepicker.min.js', array('defer' => true), 'smf_datepicker');
```

Comment:
That might be a very useful PR. 

And yes, I am aware that the proper spelling is JavaScript, but the function is named `loadJavascriptFile`, so that's what I need to use here.